src/library/Uno.Themes.WinUI.Markup/Theme/RatingControl.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Microsoft.UI.Text;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class RatingControl
+		{
+			public static class Resources
+			{
+				public static class Default
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlForeground")]
+						public static ResourceValue<Brush> Default => new("RatingControlForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlForegroundUnselected")]
+						public static ResourceValue<Brush> Unselected => new("RatingControlForegroundUnselected", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlForegroundUnselectedPointerOver")]
+						public static ResourceValue<Brush> UnselectedPointerOver => new("RatingControlForegroundUnselectedPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlForegroundSelected")]
+						public static ResourceValue<Brush> Selected => new("RatingControlForegroundSelected", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlForegroundSelectedPointerOver")]
+						public static ResourceValue<Brush> SelectedPointerOver => new("RatingControlForegroundSelectedPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlForegroundSelectedDisabled")]
+						public static ResourceValue<Brush> SelectedDisabled => new("RatingControlForegroundSelectedDisabled", true);
+					}
+
+					public static class PlaceholderForeground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlPlaceholderForeground")]
+						public static ResourceValue<Brush> Default => new("RatingControlPlaceholderForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlPlaceholderForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("RatingControlPlaceholderForegroundPointerOver", true);
+					}
+
+					public static class CaptionForeground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "RatingControlCaptionForeground")]
+						public static ResourceValue<Brush> Default => new("RatingControlCaptionForeground", true);
+					}
```

Comment:
This doesn't need a class.
```suggestion
						[ResourceKeyDefinition(typeof(Brush), "RatingControlCaptionForeground")]
						public static ResourceValue<Brush> CaptionForeground => new("RatingControlCaptionForeground", true);
```